Day-01/ruby/Taucher2003/day_01.rb

```diff
@@ -0,0 +1,29 @@
+input = ARGF.read.lines.map { |line| line.split('   ').map { |num| Integer(num, exception: true) } }
```

Comment:
What exactly does this mean, you're expected to supply the name of the input file via the command line arguments?

Reply:
It is reading from STDIN, so you can execute it with `cat input.txt | ruby day_01.rb`